iothub/service/src/Utilities/Fx.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Reflection;
+
+namespace Microsoft.Azure.Devices.Common
+{
+    internal static class Fx
```

Comment:
Moved from another dir, with everything but this method removed.

@brycewang-microsoft, this should be on your list of things to update for the exceptions work.